test/spec/test_type/schema/operations.zig

```diff
@@ -0,0 +1,169 @@
+const ssz = @import("consensus_types");
+
+pub const Phase0Operations = struct {
+    pre: ssz.phase0.BeaconState,
+    post: ssz.phase0.BeaconState,
+    attestation: ssz.phase0.Attestation,
+    attester_slashing: ssz.phase0.AttesterSlashing,
+    block: ssz.phase0.BeaconBlock,
+    deposit: ssz.phase0.Deposit,
+    proposer_slashing: ssz.phase0.ProposerSlashing,
+    voluntary_exit: ssz.phase0.SignedVoluntaryExit,
+};
+
+pub const AltairOperations = struct {
+    pre: ssz.altair.BeaconState,
+    post: ssz.altair.BeaconState,
+    attestation: ssz.altair.Attestation,
+    attester_slashing: ssz.altair.AttesterSlashing,
+    block: ssz.altair.BeaconBlock,
+    deposit: ssz.altair.Deposit,
+    proposer_slashing: ssz.altair.ProposerSlashing,
+    voluntary_exit: ssz.altair.SignedVoluntaryExit,
+    sync_aggregate: ssz.altair.SyncAggregate,
+};
+
+pub const BellatrixOperations = struct {
+    pre: ssz.bellatrix.BeaconState,
+    post: ssz.bellatrix.BeaconState,
+    attestation: ssz.bellatrix.Attestation,
+    attester_slashing: ssz.bellatrix.AttesterSlashing,
+    block: ssz.bellatrix.BeaconBlock,
+    deposit: ssz.bellatrix.Deposit,
+    proposer_slashing: ssz.bellatrix.ProposerSlashing,
+    voluntary_exit: ssz.bellatrix.SignedVoluntaryExit,
+    sync_aggregate: ssz.bellatrix.SyncAggregate,
+    body: ssz.bellatrix.BeaconBlockBody,
+};
+
+pub const CapellaOperations = struct {
+    pre: ssz.capella.BeaconState,
+    post: ssz.capella.BeaconState,
+    attestation: ssz.capella.Attestation,
+    attester_slashing: ssz.capella.AttesterSlashing,
+    block: ssz.capella.BeaconBlock,
+    deposit: ssz.capella.Deposit,
+    proposer_slashing: ssz.capella.ProposerSlashing,
+    voluntary_exit: ssz.capella.SignedVoluntaryExit,
+    sync_aggregate: ssz.capella.SyncAggregate,
+    body: ssz.capella.BeaconBlockBody,
+    execution_payload: ssz.capella.ExecutionPayload,
+    address_change: ssz.capella.SignedBLSToExecutionChange,
+};
+
+pub const DenebOperations = struct {
```

Comment:
if fields in `DenebOperations` and `DenebOperationsOut` are just the same we can deduplicate them, given a Container "Type" is just derived from Container

specifically we can create a function running at comptime `pub fn OutType(comptime T: type) type {` and pass in `DenebOperations` following the same technique here https://github.com/ChainSafe/ssz-z/blob/76cde3c7ea23b33d2789e8e372003272abe0e335/src/ssz/type/container.zig#L284

and instead of declaring `DenebOperationsOut`, we can do
```zig
const DenebOperationsOut = OutType(DenebOperations);
```